server/schemas/schemaController.js

```diff
@@ -91,6 +91,25 @@ module.exports = {
   /* READ requests */
 
   /**
+   * Request a schema template.
+   * @param {Object} request the http ClientRequest object
+   * @param {Object} response the http ServerResponse object
+   */
+  fetchTemplate: function(request, response) {
```

Comment:
Seems like the method name should be consistent – how does a "fetch*" differ from a "get*"?